src/js/base/module/Codeview.js

```diff
@@ -50,6 +50,34 @@ export default class CodeView {
     this.context.triggerEvent('codeview.toggled');
   }
 
+  /**
+   * purify code view
+   * @param value
+   * @returns {*}
+   */
+  purify(value) {
+    if (this.options.codeviewFilter) {
+      // filter code view regex
+      value = value.replace(this.options.codeviewRegex, '');
+      // allow specific iframe tag
+      var whitelist = this.options.codeviewIframeWhitelistSrc.concat(this.options.codeviewIframeWhitelistSrcOrigin);
```

Comment:
How about to rename `codeviewIframeWhitelistSrcOrigin` to `codeviewIframeWhitelistSrcBase`?